Gopkg.toml

```diff
@@ -29,3 +29,7 @@
 [[constraint]]
   name = "github.com/BurntSushi/toml"
   version = "0.3.0"
+
```

Comment:
Please update the `go.mod` file appropriately instead, as the `dep` files are only there as a relic and will be removed very soon.

Reply:
`go.mod` was updated, I just needed to get it into `dep` as well to get the travis build passing. I'll rebase once the earlier versions of go are gone and take care of the conflict.

Reply:
Have a pull request in for go 1.12/1.13/tip only travis build and have rebased.